kasa/tests/fakeprotocol_smart.py

```diff
@@ -46,6 +48,8 @@ def __init__(
                 for comp in self.info["component_nego"]["component_list"]
             }
         self.list_return_size = list_return_size
+        self.warn_fixture_missing_methods = warn_fixture_missing_methods
+        self.fix_incomplete_fixture_lists = fix_incomplete_fixture_lists
```

Comment:
Should we rather do the fix always?

Reply:
It’s only for the test for the runtime behaviour, I.e if it happens in the real world.